### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Flameygod1123** [rscwn/MobileBloxV2#4]
How do I put this into roblox? 
Do I put the shared lib into lib/ 
Then do the oncreate method from previous mobileblox version?

**Comment by Flameygod1123** [comment]
There's no arm. Also whe I try to build this in andriod studio it does not make a apk it just shows me the files

**Comment by rscwn** [comment]
> There's no arm. Also whe I try to build this in andriod studio it does not make a apk it just shows me the files

It's to be built in Visual Studio, not Android Studio, it's said in the ReadMe.md

**Comment by Flameygod1123** [comment]
How do I build it in vsc what extensions do I install and what do I click
(I know I'm starting to get annoying but I want to ry this)

**Comment by rscwn** [comment]
> How do I build it in vsc what extensions do I install and what do I click
> (I know I'm starting to get annoying but I want to ry this)

You don't use Visual Studio Code, it's just Visual Studio. Install Mobile app development with C++ ( forgot the exact name of the package but it's something similar to that. ).

I used Visual Studio 2022 to build it with the configuration Release|ARM

Use the latest NDK that is available in VS if it isn't default yet.

**Comment by Flameygod1123** [comment]
I need the package name please

**Comment by Flameygod1123** [comment]
And how to build apk (if possible)

**Comment by rscwn** [comment]
> I need the package name please 

It's on the Visuals Studio Installer

**Comment by rscwn** [comment]
> And how to build apk (if possible)

Setup a Gradle or Ant project and add a reference in it to the NativeLib/MobileBloxV2 Project.
Visual Studio 2022 has issues with Gradle and Ant though so I didn't include that though you can try with VS2017 ( Which is what the source was created in. )

**Comment by Flameygod1123** [comment]
Could you make a video of you compiling it for me to follow

**Comment by rscwn** [comment]
> Could you make a video of you compiling it for me to follow

Sorry but no, I'm not going to.

**Comment by Flameygod1123** [comment]
So what do I download in vsc and how do I build it
(Please make it detailed so I can get off your nervers)

**Comment by rscwn** [comment]
> So what do I download in vsc and how do I build it
> (Please make it detailed so I can get off your nervers)

You use Visual Studio 2022.

Download the workload **Mobile development with C++** which is available on the Visual Studio Installer.

Then open my project. ( Double click the .sln and open with VS2022 )

Once it's loaded, use Ctrl+B to build it, the library will be outputted in NativeLib/ARM/Release.

You place it in Roblox just like you would with any other cheat, just also place **libc++_shared.so** in libs/ alongside **libMobileBlox.so**.

Done.

**Comment by rscwn** [comment]
> I did ctrl b and it said build started 0 successful 1 up to date 0 failed

Not sure what happened or what to do

**Comment by Flameygod1123** [comment]
I've added libmobileblox.so and libsharedc++_.so to both armv7 and v8
And the ui doesn't show up but the console prints "done"
Should I be doing the on create method too?

**Comment by rscwn** [comment]
> I've added libmobileblox.so and libsharedc++_.so to both armv7 and v8
> And the ui doesn't show up but the console prints "done"
> Should I be doing the on create method too?

Yes, you have to loadlibrary libMobileBlox.so ( OnCreate method ).

**Comment by Flameygod1123** [comment]
How do I do it?

**Comment by rscwn** [comment]
> How do I do it?

[https://github.com/rscwn/MobileBlox]()
